### ISSUE TITLE
Issue with importing firebase modules

### ISSUE THREAD

**Issue by jeanduplessis** [ef4/ember-auto-import#25]
As mentioned in #7 the importing of inner modules seem to bring the code into vendor.js, however there seems to be a specific issue with the firebase package.

Here's a simple app that shows the issue: https://github.com/jeanduplessis/ember-auto-import-firebase

In `routes/application.js` we import both the Firebase app and messaging parts:
```
import firebase from 'firebase/app';
import 'firebase/messaging';
```

The expected behaviour is that a `messaging` function will exist in the firebase variable. (See console.log statement in activate hook).

Unfortunately this is undefined when importing the modules like so.

If I import the Firebase kitchen sink (see https://github.com/jeanduplessis/ember-auto-import-firebase/compare/kitchen-sink) then it works, but unfortunately then all the other stuff is imported as well.

If I use `ember-browserify` to import the modules then it works as well (see https://github.com/jeanduplessis/ember-auto-import-firebase/compare/ember-browserify)

**Comment by ef4** [comment]
Thanks for the complete reproduction, I see what's going on.

Both modules are getting discovered and packaged, but they're getting packaged separately from each other. Presumably `firebase/messaging` tries to import `firebase/app` in order to mutate it, but it's getting an independent copy. 

I have work underway that should fix this case. Mostly it involves making sure we only ever run Webpack once per application, with all the imported modules pulled in together simultaneously.

**Comment by ef4** [comment]
I confirmed this is fixed in 1.0.0-alpha.0.
